packages/block-editor/src/components/iframe/index.js

```diff
@@ -70,10 +73,10 @@ function bubbleEvents( doc ) {
 		}
 	}
 
-	const eventTypes = [ 'dragover', 'mousemove' ];
+	const eventTypes = [ 'dragover', 'mousemove', 'keydown' ];
 
 	for ( const name of eventTypes ) {
-		doc.addEventListener( name, bubbleEvent );
+		doc.body.addEventListener( name, bubbleEvent );
```

Comment:
I don't think this makes a difference? This could potentially ignore some events.

Reply:
Actually, this makes a lot of difference for me. If I catch the event on body I'm able stop the propagation of the React event. (This is what makes the theory works :P)